cache/remote_cache.go

```diff
@@ -8,13 +8,25 @@ import (
 	"github.com/go-kit/log/level"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/grafana/dskit/promregistry"
 )
 
 var (
 	_ Cache = (*MemcachedCache)(nil)
 	_ Cache = (*RedisCache)(nil)
 )
 
+const (
+	labelName             = "name"
+	labelBackend          = "backend"
+	backendRedis          = "redis"
+	backendMemcached      = "memcached"
+	cachePrefix           = "cache_"
+	getMultiPrefix        = "getMulti_"
```

Comment:
Pay attention that the "m" should be lowercase. I would also clarify which prefix it is.

```suggestion
	getMultiMetricNamePrefix        = "getmulti_"
```

Reply:
Thank you so much for catching this. I did multiple manual tests and this was missed from my visual observation. I will see if I can test to catch this.